Zend/zend_builtin_functions.stub.php

```diff
@@ -7,23 +7,27 @@ function func_num_args(): int {}
 /** @return mixed */
 function func_get_arg(int $arg_num) {}
 
-function func_get_args(): array|false {}
+function func_get_args(): array {}
 
 function strlen(string $str): int {}
 
 function strcmp(string $str1, string $str2): int {}
 
-function strncmp(string $str1, string $str2, int $len): int|false {}
+function strncmp(string $str1, string $str2, int $len): int {}
 
-function error_reporting($new_error_level = UNKNOWN): int {}
+function strcasecmp(string $str1, string $str2): int {}
+
+function strncasecmp(string $str1, string $str2, int $len): int {}
+
+function error_reporting($new_error_level = UNKNOWN): ?int {}
```

Comment:
Why did this become nullable?

Reply:
See lines 519 - 521 which does:
```
if (UNEXPECTED(!new_val)) {
	return;			
}
```

Maybe this is unreachable and should be an assertion but that mean that it could return null.

Reply:
https://github.com/php/php-src/commit/fbe19a6eb7d6f689508d8b2b24bfc48935deff2c :)

Reply:
welp ... need to get to know some of those macros better